signer/src/config/default.toml

```diff
@@ -82,6 +88,12 @@ network = "regtest"
 # Required: true
 deployer = "SN2V7WTJ7BHR03MPHZ1C9A9ZR6NZGR4WM8HT4V67Y"
 
+# The signer database endpoint (pgsql connection string)
+#
+# Required: true
+# Environment: SIGNER_SIGNER__DB_ENDPOINT
+db_endpoint = "postgresql://postgres:postgres@localhost:5432/signer"
```

Comment:
Can we also update the hard-coded database URL in `signer/src/testing/storage.rs` as well as the postgres service entry in `docker-compose.test.yml` to all use the same username and password as the value here? This will just make my life a little easier since I use the testing docker compose file for local development, and if this is going to be used as the database URL it's just a little easier to not have to change anything.

Reply:
Sure! Didn't see those, will do that in a little bit :+1:

Reply:
Updated in [d3b79ff](https://github.com/stacks-network/sbtc/pull/521/commits/d3b79ff5626f2537cdb8a4564f11670a13ae156d)